core/templates/dev/head/pages/about.html

```diff
@@ -426,6 +426,14 @@
   </script>
 {% endblock %}
 
+{% block footer_js %} 
+  {{ super() }}
+  <script>
+    {{ include_js_file('pages/About.js') }}
+    {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
```

Comment:
I saw this behavior of included required files again and thought it would helpful to know where the file is required.